test/functional/replace-by-fee.py

```diff
@@ -72,6 +72,10 @@ def set_test_params(self):
                            ["-mempoolreplacement=0"]]
 
     def run_test(self):
+        # Leave IBD and ensure nodes are synced
+        self.nodes[0].generate(1)
+        self.sync_all()
```

Comment:
maybe stick the sync_call after  the first make_utxo on line 79, just to be sure in the future?

Reply:
Sounds good, done.